code/style.css

```diff
@@ -1,7 +1,194 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+* {box-sizing: border-box}
+
+
+body, html {
+  height: 100%;
+  background-color: black; 
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("bakery.jpg");
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  transition: 0.4s;
+}
+
+.hero-image:hover {
+  opacity: 0.1;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+  font-family: 'Lucida Sans';
+  font-size: 40px;
+}
+
+.sticky {
+  position: fixed;
+  top: 0;
+  width: 100%
+}
+
+.sticky + .content {
+  padding-top: 100px;
+}
+
+.middle-text {
+  color: black;
+  background-color: white;
+}
+
+p {
+  text-align: center;
+  font-family: 'Lucida Sans';
+}
+
+.muffin1 img {
+float: right;
+width: 100px;
+height: 100px;
+margin-top: 10%;
+}
+
+.muffin2 img {
+  float:left;
+  width: 100px;
+  height: 100px;
+  margin-top: 10%;
+}
+
+h3 {
+  text-align: center;
+  font-family: 'Lucida Sans';
+  font-size: 70px;
+  margin-top: 0.5px;
+  
+}
+
+
+
+.content {
+  padding: 16px;
+}
+
+.sticky {
```

Comment:
Perhaps one idea could put all the styling for sticky and sticky + content together? I also noticed that you have styling for this on row 35 - 43.